scss/detail.scss

```diff
@@ -93,3 +95,63 @@
     margin-right: 12px;
   }
 }
+
+
+@media screen and (max-width: $break-medium) {
+  .book-main {
+    flex-direction: column-reverse;
+  }
+
+  .book-container {
+    flex-direction: column;
+    padding: 38px 18px 43px 18px;
+    margin-bottom: 20px;
+    max-width: 336px;
+  }
+
+  .book-cover {
+    height: 298px;
```

Comment:
height: 300px;

Reply:
In the design 298px is the exact measure